react/react-training/src/app/screens/Login/components/SignInForm/components/Validation/index.js

```diff
@@ -0,0 +1,9 @@
+import { validateEmail } from '@utils';
```

Comment:
I think that this isn't a component, so move it to `utils.js` or `validation.js` file inside of `SignInForm`